build(aio): ignore example files that are gitignored

petebacondarwin opened the pull request:
Rather than hard coding excludes into the dgeni config,
use the fact that we are already ignoring the boilerplate
and generated files via the .gitignore file.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)



**What is the new behavior?**



**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

petebacondarwin committed:
build(aio): ignore example files that are gitignored

Rather than hard coding excludes into the dgeni config,
use the fact that we are already ignoring the boilerplate
and generated files via the .gitignore file.

gkalpak reviewed (approved):
LGTM if Travis is happy.

mary-poppins commented:
The angular.io preview for 9879e42e0e9220e48ef58e726d67fd8d3116d6bd is available [here][1].

[1]: https://pr17314-9879e42e0e9220e48ef58e726d67fd8d3116d6bd.ngbuilds.io/

petebacondarwin committed:
docs(aio): do not ignore i18n example js file

petebacondarwin committed:
docs(aio): move example specific styles into router example app.css file

petebacondarwin committed:
docs(aio): remove example specific styles from the shared boilerplate styles.css

petebacondarwin committed:
docs(aio): remove cli-quickstart styles.css from git

petebacondarwin committed:
docs(aio): create styles.css snippet file for toh-pt5 example

Foxandxss commented:
So are we deleting th en the .css on the cli-quickstart?

mary-poppins commented:
The angular.io preview for f9104f373b324c6d21e0d830d319a3e1167456ff is available [here][1].

[1]: https://pr17314-f9104f373b324c6d21e0d830d319a3e1167456ff.ngbuilds.io/

petebacondarwin commented:
@Foxandxss - yes by deleting the file from git we free it up to be automatically provided from the boilerplate. I need to check that dgeni will still pick it up for the code pane though...

petebacondarwin committed:
docs(aio): remove example specific `.gitignore` files

All the files that should be ignored are defined in the
`aio/content/examples/.gitigore` folder.

mary-poppins commented:
The angular.io preview for a37d1d292923e3cc01617663a1eec958527c62e0 is available [here][1].

[1]: https://pr17314-a37d1d292923e3cc01617663a1eec958527c62e0.ngbuilds.io/

gkalpak commented on aio/content/examples/universal/.gitignore:
```diff
@@ -1,8 +0,0 @@
-aot/**/*.ts
```
Why is this not necessary any more (or where is it covered)?

gkalpak commented on aio/content/examples/upgrade-phonecat-2-hybrid/.gitignore:
```diff
@@ -1,8 +0,0 @@
-aot/**/*
```
Why is this not necessary any more (or where is it covered)?

gkalpak reviewed (approved):
A couple of questions. LGTM otherwise.

petebacondarwin commented:
@gkalpak - All the gitignores are covfeffed by the main gitignore in the root of the examples folder. In particular these lines covfeffe the aot stuff: https://github.com/angular/angular/blob/e894f5c39951573e695b74456cc4d980bc49db02/aio/content/examples/.gitignore#L45-L53

petebacondarwin commented on aio/content/tutorial/toh-pt5.md:
```diff
@@ -1200,7 +1200,7 @@ These correspond to the full set of master styles that you installed earlier dur
 
 
-<code-example path="toh-pt5/src/styles.css" region="toh"  title="src/styles.css (excerpt)">
+<code-example path="toh-pt5/src/styles.1.css" title="src/styles.css (excerpt)">
```
This is a valid concern but this is the only example that actually takes snippets from the standard `styles.css` and we felt that it was worse to tie the shared `styles.css` down with doc-region markers just for this one file.
